cookbooks/crystal_preferred_orientation_olivine_fraters_billen_2021/doc/cpo.part.prm

```diff
@@ -0,0 +1,81 @@
+#specify posstprocessor objects that will be run at the end of each time step
+#The statistics is important for cpo plotter for reading the data
+#The particles and crystal preferred orientation are required for CPO calculations
+subsection Postprocess
+  set List of postprocessors = velocity statistics, visualization, particles, crystal preferred orientation
+
+  #specify data output for visualizing in paraview or visit
+  subsection Visualization
+    set Time between graphical output = 1e5
+    set List of output variables = stress, viscosity, strain rate
```

Comment:
`Viscosity` was a deprecated option that was removed earlier this week. Please use `material properties` instead. By default `material properties` will output other quantities as well such as density. You could either leave it at the default values, or limit it to just the viscosity if you think that is important.